### PR TITLE
fix(cli): allow cli token to support non valid js keys for components

### PR DESCRIPTION
**Pull request by npirotte**
## 📝 Description

This pull request allows the usage of non JS valid tokens for components.

## ⛳️ Current behavior (updates)

```
components: {
  "design-system/Button": {}
}
```

fails to compile with SyntaxError: Property or signature expected. (242:1)

## 🚀 New behavior

Non js kind of key are supported, previous snippet will generate following output:

```
'design-system/Button': {
      sizes: never;
      variants: never;
}
```

## 💣 Is this a breaking change (Yes/No):

No impact foresaw

### CODE REVIEW

**Review by TimKolberger — COMMENTED** [review]
Great work @npirotte! ✨ Thank your for this great addition!
I just added a comment and suggestion. If this is tested, we'll go forward with this PR 👍

File: tooling/cli/src/command/tokens/extract-component-types.ts [hunk 1]
```diff
@@ -27,13 +27,19 @@ export function extractComponentTypes(theme: Record<string, unknown>) {
   )
 }
 
+function escapeComponentName(componentName: string) {
+  return componentName.match(/^[a-zA-Z0-9\-\_]+$/)
```
[the remark this answers is not in the document]
Comment (npirotte) [reply tooling/cli/src/command/tokens/extract-component-types.ts:31]: Sure, thanks for the review, will do!

**Review by TimKolberger — APPROVED** [review]
Looks good to me 🎉

**Comment by vdhpieter** [comment]
@TimKolberger What should happen to get this merged? It would really help!

**Pull request merged**